src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -116,13 +116,11 @@
 
         <!-- Cache -->
 
-        <service id="api_platform.metadata.resource.cache" parent="cache.system" public="false">
-            <tag name="cache.pool" />
-        </service>
+        <service id="api_platform.metadata.resource.cache" parent="api_platform.cache" public="false" />
 
-        <service id="api_platform.metadata.property.cache" parent="cache.system" public="false">
-            <tag name="cache.pool" />
```

Comment:
I removed this tags as they are useless when empty and because they prevent using the `ArrayAdapter` or any other having a different signature than the one expected.